### ISSUE TITLE
Automatically created role's policies do not show in UI

### ISSUE THREAD

**Issue by vtiffenberg** [instedd/cdx#800]
Correct automatically generated policies JSON (separate in different statements), so it matched the appropriate layout for it to be editable in the UI

**Comment by vtiffenberg** [comment]
![screen shot 2016-02-18 at 4 23 42 pm](https://cloud.githubusercontent.com/assets/4836220/13155510/dcfa29b2-d65c-11e5-9cad-67ea1f82c04b.png)

**Comment by bcardiff** [comment]
Actions in automatically generated policies are "*" and those seems to not be supported by the UI. cc: @matiasgarciaisaia.
Should we add an `(all)` option for mapping "*" ?

**Comment by nekron** [comment]
This was intentionally left outside the editor as it doesn't imply "all" but "all my granter has", and in this case, is always the owner of the organization. The original design had "inherit permissions from granter" which based on that, doesn't make sense.

TLDR: Agree, in this case the best thing to do is to map \* as "all".

**Comment by matiasgarciaisaia** [comment]
Support for `*` has been implemented and then dropped (in 6bd6a01776593160976b5a010baba30de628dfac) because of the reasons @nekron said.

**Comment by bcardiff** [comment]
This was not only about **All** option in action but also about displaying institution as entities to scope statement actions. User is unable to create scopes by institution, but they can view them and update them.

check notes in 41569043 for details

![screen shot 2016-02-19 at 3 47 11 pm](https://cloud.githubusercontent.com/assets/459923/13185647/08278c3c-d721-11e5-9cfc-a242651428cb.png)

**Comment by pmallol** [comment]
Ok for Institution Roles in Version: 0.11-pre3
